.github/workflows/web.yml

```diff
@@ -143,7 +148,9 @@ jobs:
 
   frontend-web-build:
     needs: [get-environment]
-    if: ${{ needs.get-environment.outputs.skip_workflow == 'false' }}
+    if: |
+      needs.get-environment.outputs.skip_workflow == 'false' &&
+      needs.get-environment.outputs.stability == 'stable'
```

Comment:
this should run in all stabilities as it is used for packaging or source delivery